Source/UI/BillingForm/View/BillingFormTextFieldView.swift

```diff
@@ -89,6 +83,17 @@ class BillingFormTextFieldView: UIView {
         updateErrorView(style: style)
     }
 
+    func refreshLayoutComponents() {
+        let handlesPhoneNumber = isPhoneNumberType()
```

Comment:
Question what does isPhoneNumberType() method do ?. Is this part of the phoneNumber kit framework.